apps/code-examples/project.json

```diff
@@ -59,7 +60,14 @@
           "namedChunks": true
         }
       },
-      "defaultConfiguration": "production"
+      "defaultConfiguration": "production",
+      "dependsOn": [
+        "^build",
+        {
+          "target": "build",
+          "projects": "manifest"
```

Comment:
We need `manifest` to build before serving the code examples SPA because it reads from the `dist` folder.